common/vpcclient/vpcfilevolume/expand_volume_test.go

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2021 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package vpcfilevolume ...
+package vpcfilevolume_test
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/IBM/ibmcloud-volume-file-vpc/common/vpcclient/models"
+	"github.com/IBM/ibmcloud-volume-file-vpc/common/vpcclient/riaas/test"
+	"github.com/IBM/ibmcloud-volume-file-vpc/common/vpcclient/vpcfilevolume"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+)
+
+func TestExpandVolume(t *testing.T) {
+	// Setup new style zap logger
+	logger, _ := GetTestContextLogger()
+	defer logger.Sync()
+
+	testCases := []struct {
+		name string
+		// Response
+		status  int
+		content string
+		// Expected return
+		expectErr string
```

Comment:
can we have expectedOutput instead-of status and what is the input considered here?

Reply:
This function returns the "models.Share". It's convenient to check the volumeId, size of the returned "Share" as compared to comparing the whole "Share" struct.

Reply:
Enhance